### PR TITLE
Add AddThingsToDoAd example

### PR DESCRIPTION
**Pull request by jradcliff**

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/travel/AddThingsToDoAd.java [hunk 1]
```diff
@@ -0,0 +1,323 @@
+// Copyright 2018 Google LLC
```
Comment (devchas) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/travel/AddThingsToDoAd.java:1]: Nit: 2023

**Review by devchas — APPROVED** [review]

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/travel/AddThingsToDoAd.java (hunk 1 above)
Comment (jradcliff) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/travel/AddThingsToDoAd.java]: Fixed

**Comment by jradcliff** [comment]
Apologies for not clarifying before, but @fiboknacky , you can just review these 2 commits:

https://github.com/googleads/google-ads-java/pull/717/files/384f9b2f8712aa6c8a612a57457c5d07ae09f30b..15bc11b65d981ed47fa268e62ff416454a4cf277

The other commits are under review in https://github.com/googleads/google-ads-java/pull/716.

**Comment by jradcliff** [comment]
Merged `main` into this PR's branch to make reviewing easier

**Comment by fiboknacky** [comment]
I'm reviewing it now. Just FYI.

**Review by fiboknacky — APPROVED** [review]

**Pull request merged**